chart/values.yaml

```diff
@@ -457,6 +457,8 @@ controlPlane:
     overwriteManifests: ""
     # OverwriteConfig can be used to overwrite the coredns config
     overwriteConfig: ""
+    
+    priorityClassName: null
```

Comment:
You have to add this field under [`config.go`](https://github.com/loft-sh/vcluster/blob/main/config/config.go#L937-L955) and [`values.yaml`](https://github.com/loft-sh/vcluster/blob/main/config/values.yaml#L255) and then run `go run ./hack/schema/main.go`

Reply:
fixed